src/main/java/io/quarkus/bot/PingWhenNeedsTriageRemoved.java

```diff
@@ -33,7 +34,10 @@ void pingWhenNeedsTriageRemoved(@Issue.Unlabeled GHEventPayload.Issue issuePaylo
         GHIssue issue = issuePayload.getIssue();
         GHLabel removedLabel = issuePayload.getLabel();
 
-        if (!removedLabel.toString().equals(Labels.TRIAGE_NEEDS_TRIAGE)) {
+        if (!Labels.TRIAGE_NEEDS_TRIAGE.equals(removedLabel.getName())) {
+            return;
+        }
+        if (issue.getLabels().isEmpty()) {
```

Comment:
Let's evacuate this corner case too: if we don't have other labels, no need to do any work.